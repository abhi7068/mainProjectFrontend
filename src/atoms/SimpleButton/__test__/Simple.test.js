import React from 'react';

import Simple from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<Simple />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<Simple />);
  expect(asFragment(render(<Simple />))).toMatchSnapshot();
});
