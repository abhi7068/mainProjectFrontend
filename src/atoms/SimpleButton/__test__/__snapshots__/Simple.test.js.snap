// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
